irods/parallel.py

```diff
@@ -293,19 +307,27 @@ def _io_part (objHandle, range_, file_, opr_, mgr_, thread_debug_id = '', queueO
 
 
 def _io_multipart_threaded(operation_ , dataObj_and_IO, replica_token, hier_str, session, fname,
-                           total_size, num_threads = 0, **extra_options):
+                           total_size, num_threads, **extra_options):
     """Called by _io_main.
     Carve up (0,total_size) range into `num_threads` parts and initiate a transfer thread for each one."""
     (D, Io) = dataObj_and_IO
```

Comment:
is there a word that better describes what `D` represents?

Reply:
Yep , it'll be the data object.  Will make it more descriptive.